proto/desmos/profiles/v1beta1/query.proto

```diff
@@ -37,23 +37,26 @@ service Query {
     option (google.api.http).get = "/desmos/profiles/v1beta1/params";
   }
 
-  // UserRelationships queries the relationships for the user having the given
-  // address
-  rpc UserRelationships(QueryUserRelationshipsRequest)
-      returns (QueryUserRelationshipsResponse) {
+  // Relationships queries all relationships
+  // It queries the relationships for the given user if the user address is
+  // provided
```

Comment:
```suggestion
  // Relationships queries all relationships for the given user, if provided. 
  // Otherwise, it queries all the relationships stored.
```